Topl/Bifrost
# CLI: Init Mainnet Command

## Purpose
- Launching a mainnet is decentralized and does not use a common seed like the testnets would
- Instead, the mainnet is initialized using registration transactions constructed using the `register` command
## Approach
- Implement InitMainnetCommand
- Extract common functionality from InitTestnetCommand
## Testing
- Verified using the procedure
  - Run `register` command using a mainnet LockAddress (i.e. `mtetmain1y3LYktMBTcdh6i3aaAKafMiFvFVgKw8Z2SYt3xCZvL9WhkBhpN4`)
  - Use the registration transaction in the `init-mainnet` command
  - Expected files are created
## Tickets
- #BN-1121
